### PR TITLE
testing with jest and react testing library done

### PR DESCRIPTION
**Pull request by MikeSC27**

### CODE REVIEW

File: src/App.test.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
...
 import App from './App';
+import { BrowserRouter } from "react-router-dom"
 
 test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+  render(    <BrowserRouter>
+    <App />
+    </BrowserRouter>);
```
Comment (elysemontano) [inline src/App.test.js]: Check your indentation

File: src/components/Footer.test.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import { BrowserRouter } from "react-router-dom"
+
+
+describe("<Footer />", () => {
+  it("renders without crashing", () => {
+    const div = document.createElement("div")
+    render(              <BrowserRouter>
+        <Footer />
+      </BrowserRouter>,
+      div)
```
Comment (elysemontano) [inline src/components/Footer.test.js]: Check indentation

**Review by elysemontano — COMMENTED** [review]
Let's clean up some of the indentation in App.test.js and Footer.test.js and I will approve.

File: src/App.test.js (hunk 1 above)
Comment (MikeSC27) [reply src/App.test.js]: corrected indentations and re-pushed.

File: src/components/Footer.test.js (hunk 2 above)
Comment (MikeSC27) [reply src/components/Footer.test.js]: corrected indentations and re-pushed.

**Review by elysemontano — APPROVED** [review]

**Pull request merged**